npm/private/npm_translate_lock_generate.bzl

```diff
@@ -668,6 +656,41 @@ def _gen_npm_import(rctx, _import, link_workspace):
         maybe_exclude_package_contents = maybe_exclude_package_contents,
     )
 
+def _generate_npm_link_targets(links_targets_bzl):
```

Comment:
Now that `links_targets_bzl` is built up in a nicer way (it is metadata about which links belong to which packages) it is easier to put the code-generation for `npm_link_targets` into an isolated util.

See the `npm/private/test/snapshots/npm_defs.bzl` diff to notice the generated code has hardly changed.